source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -110,6 +113,40 @@ namespace aspect
       return phi_theta_d;
     }
 
+    template <int dim>
+    Point<3>
+    EllipsoidalChunk<dim>::EllipsoidalChunkGeometry::push_forward_topography(const Point<3> &phi_theta_d_hat) const
+    {
+      const double d_hat = phi_theta_d_hat[2]; // long, lat, depth
+      Point<dim-1> phi_theta;
+      const double rad_to_degree = 180/numbers::PI;
+      if (dim == 3)
+        phi_theta = Point<dim-1>(phi_theta_d_hat[0] * rad_to_degree,phi_theta_d_hat[1] * rad_to_degree);
+      const double h     = topography->value(phi_theta);
```

Comment:
It seems you use the `topography` member only in this line and the corresponding pull... function. Have you considered handing over the reference as a parameter? This would remove the member variable that just copies an address from simulator. Since this is only an initial condition and reading and copying an address is fast that might be the easier solution. Also no need to worry about initialization then.